test/tools/yulInterpreter/Interpreter.h

```diff
@@ -62,7 +62,17 @@ enum class LoopState
 
 struct InterpreterState
 {
-	dev::bytes calldata;
+	// 64 bytes of random calldata
+	dev::bytes calldata = {
```

Comment:
It might be better to do that in the fuzzer. If you instantiate a fresh interpreter state, you might expect empty calldata. It is different with the other default values, because they have to return something (i.e. are just values and not dynamically-sized data structures).

Reply:
I moved the initialization of calldata to a wrapper used only by the fuzzer